### DISCUSSION TITLE
Allow uploading valid rows in the WorkBench and create a new data set with the errored ones

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5845] [Feature Requests]
It would be incredibly useful to give users the option to upload a batch of cleared records and create a new `spdataset` record with the remaining errored ones. 

See how TaxonWorks' batch import tool imports valid records and shows errored ones to solve: 
<img width="840" alt="Screenshot 2023-07-12 at 1 05 46 PM" src="https://github.com/specify/specify7/assets/37256050/3ed59de9-c5ce-4162-8d87-08520d4968f3">

**Comment by maxpatiiuk** [comment]
This is a duplicate of #2054. Closing that one in favor of this one

**Comment by maxpatiiuk** [comment]
> During the conversion process, it would be quite useful to be able to upload a data set, and instead of failing on rows with errors, have the WorkBench skip the rows with errors.
>
> Then, there should be an option to create a data set of just error rows, which can then be sent back to the institution for correction.
>
> Suggested by @grantfitzsimmons
